pycbc/types/array.py

```diff
@@ -615,6 +615,19 @@ def conj(self):
     def squared_norm(self):
         """ Return the elementwise squared norm of the array """
 
+
+    # Adding the @_checkother decorator here raises a bizarre failure when
+    # calling weave. KeyError: 'V'. Why??
```

Comment:
Ah, right. It will return a subclass of numpy.ndarray (AlignedArray or something like that). For some reason scipy doesn't like that type, even though it is a simply subclass. I'll have to think about this. This issue crops up in many places actually, and it would be good to have the "right" fix.